samples/bot-calling-meeting/csharp/Source/CallingBotSample/Services/MicrosoftGraph/OnlineMeetingService.cs

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using CallingBotSample.Options;
+using Microsoft.Extensions.Options;
+using Microsoft.Graph;
+
+namespace CallingBotSample.Services.MicrosoftGraph
+{
+    public class OnlineMeetingService : IOnlineMeetingService
+    {
+        private readonly GraphServiceClient graphServiceClient;
+        private readonly UsersOptions usersOptions;
+
+        public OnlineMeetingService(GraphServiceClient graphServiceClient, IOptions<UsersOptions> usersOptions)
+        {
+            this.graphServiceClient = graphServiceClient;
+            this.usersOptions = usersOptions.Value;
+        }
+
+        /// <inheritdoc/>
+        public Task<OnlineMeeting> Create(string subject, params string[] participantsIds)
```

Comment:
Don't use `params` here-- the call site will be harder to understand. Even worse if you ever need to add more parameters to Create. (Doesn't look like you use the params functionality anyway)

Also consider changing these to IEnumerable<string>, so that at the callsite you don't need to call ToArray().